cmd/cql/internal/generate.go

```diff
@@ -57,10 +57,16 @@ or input a passphrase by
 	DebugFlag:  flag.NewFlagSet("Debug params", flag.ExitOnError),
 }
 
+const (
+	TestnetCN     = "cn"
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported const TestnetCN should have comment (or a comment on this block) or be unexported